### PR TITLE
Update id to match page template

### PR DESCRIPTION
**Pull request by IanDelMar**
For consistency, adds a page template specific ID for the wrapper in the no-title page template.

### CODE REVIEW

File: page-templates/fullwidthpage.php [hunk 1]
```diff
@@ -16,9 +16,14 @@
...
 }
+
+$wrapper_id = 'full-width-page-wrapper';
+if ( is_page_template( 'page-templates/no-title.php' ) ) {
+	$wrapper_id = 'no-title-page-wrapper';
+}
 ?>
 
-<div class="wrapper" id="full-width-page-wrapper">
+<div class="wrapper" id="<?php echo $wrapper_id; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- ok. ?>">
```
Comment (bacoords) [inline page-templates/fullwidthpage.php:26]: What's the reasoning for not just using `esc_attr()` here?

**Review by bacoords — COMMENTED** [review]
Do we need to also match the HTML comment on line 51?

File: page-templates/fullwidthpage.php (hunk 1 above)
Comment (IanDelMar) [reply page-templates/fullwidthpage.php:26]: We know that we will never have to escape the attribute. It's either `'full-width-page-wrapper'` or `'no-title-page-wrapper'`. Calling `esc_attr()` just adds a call without any benefit.

**Comment by IanDelMar** [comment]
> Do we need to also match the HTML comment on line 51?

Updated the HTML comment

**Review by bacoords — APPROVED** [review]

**Pull request merged**